embedded_hub.go

```diff
@@ -55,6 +55,24 @@ func StartHub(bootstrapAddress *C.char, bootstrapAddressLen C.int, privKey unsaf
 		ProcessMessage)
 }
 
+//export TestStartHub
```

Comment:
Why we have this function here? Can we place this functionality in the testhub.go?

Reply:
Done